virt/virt-2-4-release-notes.adoc

```diff
@@ -109,10 +109,14 @@ from being inadvertently started before an upload has completed.
 [id="virt-2-4-known-issues"]
 == Known issues
 //https://bugzilla.redhat.com/show_bug.cgi?id=1855067 - Requested doc text for this defect.
-//https://bugzilla.redhat.com/show_bug.cgi?id=1835426 - Requested doc text for this defect.
-//https://bugzilla.redhat.com/show_bug.cig?=id=1959237
+
 * If you upgrade to {VirtProductName} {VirtVersion}, both older and newer versions of common templates are available for each combination of operating system, workload, and flavor. When you create a virtual machine by using a common template, you must use the newer version of the template. Disregard the older version to avoid issues. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1859237[*BZ#1859237*])
 
+* In this release, if a name is specified for a virtual machine instance (VMI) in the VMI template associated with the virtual machine (VM), it is ignored. The names for the VMI and VM are matched correctly.
+
+If a name is specified for a virtual machine instance (VMI) in the VMI template for the virtual machine (VM), it is ignored. The
```

Comment:
Is this sentence  intentionally there too? The sentence above looks descriptive enough. Maybe move just the link up and remove the rest.

Reply:
@rmohr The sentences on Line 117 and 188 are redundant, removing them.